emperor/support_files/css/emperor.css

```diff
@@ -102,7 +98,7 @@ div.emperor-separator:hover {
 div#emperor-menu {
   height: 100%;
   font-family: Helvetica, sans-serif;
-  width: 26%;
+  width: 27%;
```

Comment:
Though note that this will be coming back.

On (Aug-18-15|17:42), Daniel McDonald wrote:
> > @@ -102,7 +98,7 @@ div.emperor-separator:hover {
> >  div#emperor-menu {
> >    height: 100%;
> >    font-family: Helvetica, sans-serif;
> > -  width: 26%;
> > +  width: 27%;
> 
> gotta bring back the classic
> 
> ![](http://i.imgur.com/cqMnINb.gif)
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/emperor/pull/407/files#r37370273